tools/demo_discovery.py

```
@@ -0,0 +1,150 @@
+    # Étape 1: Découverte automatique
+    print("\n📡 ÉTAPE 1: Découverte automatique des champs...")
+    
+    # Fichier temporaire pour la config découverte
+    temp_dir = Path(tempfile.mkdtemp()) if args.keep_temp else Path(tempfile.mkdtemp())
+    discovered_config = temp_dir / "discovered_config.json"
+    
+    # Lancer la découverte
+    discover_cmd = f'python discover_fields.py "{args.google_photos_dir}" --output "{discovered_config}" --summary'
+    print(f"   Commande: {discover_cmd}")
+    
+    os.system(discover_cmd)
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)